lib/src/view/analysis/retro_screen.dart

```diff
@@ -214,10 +184,10 @@ class _RetroAnalysisBoardState
   AnalysisPrefs get analysisPrefs => ref.watch(analysisPreferencesProvider);
 
   @override
-  bool get showAnnotations => analysisPrefs.showAnnotations;
+  bool get showAnnotations => false;
 
   @override
-  bool get hideBestMoveArrow => analysisState.isSolving;
+  bool get hideBestMoveArrow => true;
```

Comment:
Hmm, don't we want to show the arrows after the solution has been found, in case the user wants to analyse further?

Reply:
I think it is better to keep it separate from analysis, and then we don't want to use analysis settings. So I found it simpler to remove the lines and arrows for now.